### PR TITLE
Update tiflash usage

### PR DESCRIPTION
**Pull request by zanmato1984**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -141,24 +148,19 @@ Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine
     set SESSION tidb_isolation_read_engines = "逗号分隔的 engine list";
     ```
 
-    会话级别的默认配置继承自 TiDB 实例级别的配置，见 2。
-
-2. TiDB 实例级别，即 INSTANCE 级别，和会话级别配置是交集关系。比如实例级别配置了 "tikv, tiflash"，而会话级别配置了 "tikv"，则只会读取 TiKV。
+    会话级别的默认配置继承自 TiDB 实例级别的配置。
 
-    在 TiDB 的配置文件添加如下配置项：
+最终的 engine 配置是实例级别和会话级别两者配置的交集。比如实例级别配置了 "tikv, tidb, tiflash"，而会话级别配置了 "tiflash"，则只会读取 TiFlash 副本。当 engine 配置为 "tikv, tiflash"，即可以同时读取 TiKV 和 TiFlash 副本，优化器会自动选择。
```
Comment (lzmhhh123) [inline tiflash/use-tiflash.md:153]: 实例级别和会话级别不取交集，使用会话级别。

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -141,24 +148,19 @@ Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine
     set SESSION tidb_isolation_read_engines = "逗号分隔的 engine list";
     ```
 
-    会话级别的默认配置继承自 TiDB 实例级别的配置，见 2。
-
-2. TiDB 实例级别，即 INSTANCE 级别，和会话级别配置是交集关系。比如实例级别配置了 "tikv, tiflash"，而会话级别配置了 "tikv"，则只会读取 TiKV。
+    会话级别的默认配置继承自 TiDB 实例级别的配置。
```
Comment (lzmhhh123) [inline tiflash/use-tiflash.md:151]:
```suggestion
    **会话级别的默认配置继承自 TiDB 实例级别的配置。**
```

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -119,13 +117,22 @@ explain analyze select count(*) from test.t;
...
+Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine 的副本，可选 engine 为 "tikv"、"tidb" 和 "tiflash"（其中 "tidb" 表示 TiDB 内部的内存表区，主要用于存储一些 TiDB 系统表，用户不能主动使用），分别有 2 个配置级别：
+
+1. TiDB 实例级别，即 INSTANCE 级别。在 TiDB 的配置文件添加如下配置项：
+
+    ```
+    [isolation-read]
+    engines = ["tikv", "tidb", "tiflash"]
+    ```
+
+    实例级别的默认配置为 `["tikv", "tidb", "tiflash"]`。
```
Comment (lzmhhh123) [inline tiflash/use-tiflash.md:133]:
```suggestion
    **实例级别的默认配置为 `["tikv", "tidb", "tiflash"]`。**
```

File: tiflash/use-tiflash.md [hunk 4]
```diff
@@ -141,24 +148,19 @@ Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine
...
 
-    ```
-    [isolation-read]
-    engines = ["tikv", "tiflash"]
-    ```
-
-    实例级别的默认配置为 `["tikv", "tiflash"]`。
+> **注意：**
+>
+> 由于 dashboard 等组件需要读取一些存储于 TiDB 内存表区的系统表，因此建议实例级别 engine 配置中始终加入 "tidb" engine。
```
Comment (yikeke) [inline tiflash/use-tiflash.md:157]:
```suggestion
> 由于 TiDB Dashboard 等组件需要读取一些存储于 TiDB 内存表区的系统表，因此建议实例级别 engine 配置中始终加入 "tidb" engine。
```

**Pull request merged**